components/lists/lists.html

```diff
@@ -0,0 +1,160 @@
+                            class="fas fa-user-tie s-icons"></i> Avatar</a></li>
+                <li class="sidebar-item"><a href="/components/alert/alert.html"><i
+                            class="fas fa-exclamation-circle s-icons"></i> Alert</a></li>
+                <li class="sidebar-item"><a href="/components/badges/badges.html"><i
+                            class="far fa-id-badge s-icons"></i> Badge</a></li>
+                <li class="sidebar-item "><a href="/components/buttons/buttons.html"><i
+                            class="fas fa-mouse s-icons"></i> Button</a></li>
+                <li class="sidebar-item"><a href="/components/cards/cards.html"><i
+                            class="fas fa-address-card s-icons"></i> Card</a></li>
+                <li class="sidebar-item"><a href="/components/image/image.html"><i class="far fa-images s-icons"></i>
+                        Image</a></li>
+                <li class="sidebar-item"><a href="/components/input/input.html"><i class="far fa-keyboard s-icons"></i>
+                        Input</a></li>
+                <li class="sidebar-item"><a href="/components/text-utilities/text-utilities.html"><i
+                            class="fas fa-heading"></i> Text Utilities</a></li>
+                <li class="sidebar-item active-item"><a href="/components/lists/lists.html"><i
+                            class="fas fa-list-ul"></i> Lists</a>
+                </li>
+                <li class="sidebar-item"><a href="/components/navigation/navigation.html"><i class="far fa-compass"></i>
+                        Navigation</a></li>
+                <li class="sidebar-item"><a href="/components/modal/modal.html"><i class="far fa-square"></i> Modal</a>
+                </li>
+                <li class="sidebar-item"><a href="/components/rating/rating.html"><i class="far fa-star"></i> Rating</a>
+                </li>
+                <li class="sidebar-item"><a href="/components/toast/toast.html"><i class="fas fa-bread-slice"></i>
+                        Toast</a></li>
+                <li class="sidebar-item"><a href="/components/grid/grid.html"><i class="fas fa-grip-horizontal"></i>
+                        Grid</a></li>
+            </ul>
+
+        </div>
+
+
+        <div id="buttons">
+            <div class="component-flex">
+                <div class="component-info">
+                    <h1>Lists</h1>
+                    <p class="txt-m bd-btm mb-2">
+                        Various forms of Lists
+                    </p>
+                </div>
+                <div class="component-display">
+                    <h2 class="mb-1">Spaced (normal lists)</h2>
+                    <div class="display-container flex-icons mb-2">
+                        <div class="list-component ">
+                            <div class="list-title txt-s ">Category</div>
+                            <div class="divider"></div>
```

Comment:
instead of using empty div, you can use the class in the upper div

Reply:
done